scanomatic/data_processing/pheno/save.py

```diff
@@ -127,7 +130,10 @@ def save_state(
         or not os.path.isfile(p)
         or _do_ask_overwrite(p)
     ):
-        np.save(p, settings.serialized())
+        with open(p, 'w') as fp:
+            json.dump(json.loads(
+                jsonizer.dumps(asdict(settings)),
+            ), fp)
```

Comment:
Should only be this
```suggestion
        jsonizer.dumps(settings, p)
```

Reply:
I assume you mean `jsonizer.dump`. I'll see if I can get that to work, it would be much neater. Should it have the default value `False` on `overwrite` though?

Reply:
overwrite is an old name and a bit confusing. it really means merge into

Reply:
and it should be `False`?

Reply:
yes I belive so, False means it just dumps the content to the target location independent of what is there, 'overwrite' means it tries to merge new info with existing info...

Reply:
I'll rename it as well then.

Reply:
fixed